File: src/main/java/ru/igrey/dev/entity/DealEntity.java
package ru.igrey.dev.entity;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@ToString
@Getter
@Setter
@AllArgsConstructor
public class DealEntity {
    private Long id;
    private Long authorId;
    private Long dealStatusId;
    private Long mikId;
    private Long officeId;
    private Long typeId;
    private Boolean archived;
}

File: src/main/kotlin/ru/igrey/dev/entity/DealEntity.kt
package ru.igrey.dev.entity

import java.time.LocalDateTime

data class DealEntity(
    val id: Long,
    val authorId: Long,
    val dealStatusId: Long,
    val mikId: Long,
    val officeId: Long,
    val typeId: Long,
    val archived: Boolean,
    val createdTime: LocalDateTime
)